### PR TITLE
DG Update Model and AddCommand

### PR DESCRIPTION
**Pull request by cowlinn**
Model is updated with a new object diagram

Add Command is updated with an accompanying Sequence Diagram

### CODE REVIEW

File: docs/diagrams/AddEventActivityDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@startuml
+start
+:User executes AddEventCommand;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+if () then
```
Comment (ReubenChay) [inline docs/diagrams/AddEventActivityDiagram.puml:8]: very nice and concise diagram!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -232,6 +275,19 @@ The following activity diagram summarizes what happens when a user executes a ne
   * Pros: Will use less memory (e.g. for `delete`, just save the person being deleted).
   * Cons: We must ensure that the implementation of each individual command are correct.
 
+
+**Aspect: Should events that occured in the past be auto-deleted on startup of app?:**
```
Comment (ReubenChay) [inline docs/DeveloperGuide.md:279]: nicely and succinctly explained our dilemma! very nice

**Review by ReubenChay — COMMENTED** [review]
LGTM!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -126,7 +126,22 @@ The `Model` component,
...
+
+
+<img src="images/ModelEventClassDiagram.png"  width="500"/>
+
+### Event Component
+
+An extension to the current Model is the `Event` Component,
+* Stores all information related to Event Objects, all of which are contained inside a `UniqueEventList` object.
+
+* Stores the current `Event` objects. Upon filtering via other commands (e.g: `calendar`), it is exposed to outsiders as an unmodifiable `ObservableList<Event>` that can be observed.
```
Comment (Ssatu) [inline docs/DeveloperGuide.md:138]: Nice noting that it is an unmodifiable list and hence defensively designed.

This is useful for reference in creating similar classes and for safe extensibility of the `Event` class!

File: docs/diagrams/AddEventActivityDiagram.puml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    :Throw InvalidCommandException;
+else ([ValidSyntax])
+if () then
```
Comment (Ssatu) [inline docs/diagrams/AddEventActivityDiagram.puml:11]: Appreciate the decision to leave out the `[Else]` condition! Improves readability of the diagram.

Good job!

**Review by Ssatu — APPROVED** [review]
Overall, LGTM!

It is unfortunate that the syntax of plantUML causes some inconvenience in readability but as a whole the diagrams remain very easy to understand!

**Pull request merged**